File: gpu/openacc/basics/exercises/OpenACC-ULHPC-Questions/Hello_World_OpenACC.f90
!! Authour: Ezhilmathi Krishnasamy (ezhilmathi.krishnasamy@uni.lu)

subroutine Print_Hello_World()
  integer ::i
  !$acc ......????? 
  do i=1,150
     print *, "hello world"
  end do
  !$acc ......???? 
end subroutine Print_Hello_World

program main
  !!use OPENACC LIBRARY ???? 
  implicit none
  call Print_Hello_World()
end program main

File: gpu/openacc/basics/exercises/OpenACC-ULHPC-Questions/Hello_World_OpenACC.c
// Authour: Ezhilmathi Krishnasamy (ezhilmathi.krishnasamy@uni.lu)

#include <stdio.h>              
// INCLUDE OPENACC LIBRARY ????

void Print_Hello_World()    
{
#pragma acc ..............?????
  for(int i = 0; i < 5; i++)
    {                                
      printf("Hello World!\n");
    }
} 

int main()
{ 
  Print_Hello_World();     
  return 0;     
}
